README.md

```diff
@@ -2,15 +2,16 @@
 
 This repository is used for creating a proof of concept around the `PolicyProvider` interface.
 
-## Task
+## Project Setup
 
-- [ ] Fork this repository and implement the `PolicyProvider`'s `Login` method against a web property of your choice using one of the scraping libraries listed below. 
-- [ ] Implement tests that accept environment variables for the credentials (login & password). 
-- [ ] Document your code and usage instructions.
-- [ ] Send in a pull request for code review.
+Before attempting to run any tests provided within this repo, please copy the file `.env.example` (new file should be named `.env`) and input valid Netflix account credentials in corresponding fields. This new file (`.env`) is used by the repo Makefile
+to load testing credentials.
```

Comment:
Cool technique

Reply:
Thanks! Its not really necessary for the basic case, but I think it's really helpful for onboarding & project-setup on larger projects.